dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -688,6 +688,10 @@ warning.package.root.invalid=CWWKE0947W: A --server-root option with an empty ar
 warning.package.root.invalid.explanation=The user performed a package request with a --server-root option specifying an empty argument and Product Extensions were detected. The default server root of wlp will be used.
 warning.package.root.invalid.useraction=No action is required.
 
+warning.package.root.invalid.runnable=CWWKE0948W: A --server-root option with --include=runnable is not an allowed combination. The default argument of wlp will be used for the server root.
+warning.package.root.invalid.runnable.explanation=The user performed a package request with a --server-root option and --include=runnable which is not an allowed combination. The default server root of wlp will be used.
```

Comment:
"The default server root of wlp will be used" should be "is used". Also there should be a comma after runnable here